server/protocols/jmap/src/test/java/org/apache/james/jmap/model/MessagePreviewGeneratorTest.java

```diff
@@ -64,4 +64,12 @@ public void computeShouldReturnStringIsLimitedTo256Length() throws Exception {
             .hasSize(MessagePreviewGenerator.MAX_PREVIEW_LENGTH)
             .isEqualTo(expected);
     }
+
+    @Test
+    public void computeShouldReturnNormalizeSpaceString() throws Exception {
```

Comment:
I changed the input for that which contain all white space. Is that enough or does it need to separate them all?